src/caret_analyze/runtime/path.py

```diff
@@ -299,6 +299,23 @@ def summary(self) -> Summary:
         """
         return self._value.summary
 
+    @property
+    def callbacks(self) -> List[CallbackBase]:
+        """
+        Get callbacks.
+
+        Returns
+        -------
+        List[CallbackBase]
+            callbacks in all nodes that comprise the node path.
+
+        """
+        callbacks = [comm.publish_node.callbacks
```

Comment:
Could you check mypy erros?
I think Util.flatten is good one.
https://github.com/tier4/CARET_analyze/blob/main/src/caret_analyze/common/util.py#L34
![image](https://user-images.githubusercontent.com/19860128/193216691-51fc9dd0-6e38-466b-81eb-4e3df39cba55.png)